### ISSUE TITLE
make serveClient option configurable

### ISSUE THREAD

**Issue by joonas-lahtinen** [balderdashy/sails-hook-sockets#4]
Currently the option is hardcoded in `lib/initialize.js`:

```javascript
serveClient: false // if anyone needs this for a good reason, let Mike know on Twitter.
```
I would consider having the option as `app.config.sockets.serveClient` to be the fastest way of acquiring the non-minified version of socket.io library for debugging.

**Comment by sgress454** [comment]
I'm fine with this, want to submit a PR?

**Comment by mikermcneil** [comment]
@joonas-lahtinen ok, I added this for you in 25319b3ac6593c73d317934a713d1b5145cceff9

Would you please update [this section of sails-docs](https://github.com/balderdashy/sails-docs/blob/master/reference/sails.config/sails.config.sockets.md#advanced-configuration) with the new config? It is `sails.config.sockets.serveClient` and it defaults to `false`.

**Comment by mikermcneil** [comment]
@joonas-lahtinen tests here: a936347b6e217b1314328d338f03e224aa871c3d

**Comment by mikermcneil** [comment]
@joonas-lahtinen nvm just did it in https://github.com/balderdashy/sails-docs/commit/b8680c5d10f214986d491bd91565968a02bacb0e

**Comment by mikermcneil** [comment]
@joonas-lahtinen @sgress454 published as `sails-hook-sockets@0.11.21`

**Comment by joonas-lahtinen** [comment]
Thanks, sorry for inactivity on this!
